UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Fulfilment.UnitTests/Helpers/FileSystemHelperTests.cs

```diff
@@ -53,5 +52,37 @@ public void Does_CreateFolder_Completed_When_Directory_Doesnot_Exists()
             A.CallTo(() => _fakefileSystem.Directory.CreateDirectory(filePath))
                             .MustHaveHappenedOnceExactly();
         }
+
+        [Test]
+        public void Does_GetFileMD5_Returns_FileDetails_With_Hash()
+        {
+            List<Common.Models.Fss.Request.FileDetail> fileDetails = new();
```

Comment:
The FileDetail qualifier Common.Models.Fss.Request. is redundant and can be removed.

Reply:
Removed